manifest.php

```diff
@@ -33,10 +33,10 @@
     'label' => 'QTI Portable Custom Interaction',
     'description' => '',
     'license' => 'GPL-2.0',
-    'version' => '4.9.1',
+    'version' => '5.0.0',
```

Comment:
Is this version correct? Do we need a major version change for this change?

Reply:
tbh I'm not sure, I just wonder what if undeclared class produce warnings.

Reply:
I actually suppose it's safe in this case to make this a major version update, since the required tao core version is updated too. so we can leave it like this